infra/k8s/README.md

```diff
@@ -78,6 +78,9 @@ Where:
 
 You might want to add them to your `rc` file (`.zshrc`, `.bashrc`, etc.), or to an `.env.sh` file that you source.
 
+In addition to the initial cluster setup, these variables should be accessible to the daemon. If these variables are
```

Comment:
This is not ideal, because now we have config in `.env.toml` and also in environment variables, and all are required. We should consolidate config for `testground` at one place - either `.env.toml` or environment variables, but having both required is not user-friendly.

WDYT?

Reply:
Currently this seems to be a problem for `kops` and `kubectl` (since we use default context there, and this has been fine).

Wouldn't `kops validate cluster --name <NAME>` solve this, if we pick up NAME (cluster name) from `.env.toml` ? Then we need to make sure `.env.toml` is configured correctly, but we do that anyways (for example for `region`). At least we wouldn't expect users to also have to set environment variables.

Reply:
@nonsense mind opening a separate issue for that?

Reply:
I agree we need to converge.

Reply:
This diff just describes the status quo.